include/baresip.h

```diff
@@ -808,7 +808,7 @@ enum ua_event {
 	UA_EVENT_CALL_ESTABLISHED,
 	UA_EVENT_CALL_CLOSED,
 	UA_EVENT_CALL_TRANSFER,
-	UA_EVENT_CALL_BLIND_TRANSFER,
+	UA_EVENT_CALL_REDIRECT,
```

Comment:
this breaks the API, perhaps better to just add a new enum value ...

Reply:
If UA_EVENT_CALL_BLIND_TRANSFER (which is not currently used in baresip project) cannot be renamed, then it means that two events will need to be sent: UA_EVENT_CALL_BLIND_TRANSFER and UA_EVENT_CALL_REDIRECT.